core/src/main/kotlin/com/alecstrong/sql/psi/core/PredefinedTable.kt

```diff
@@ -0,0 +1,8 @@
+package com.alecstrong.sql.psi.core
+
+data class PredefinedTable(
+  val name: String,
+  val content: String,
+) {
+  val fileName = "___predefined/$name"
```

Comment:
Added because we could clash with user defined file names and we check the file name if we need to use the base package name in sqldelight. This isn't a real solution, but a doable and easy workaround.

Reply:
Maybe we could exclude the file from the file index instead? Or somehow create a temp file/remove the virtual file but keep the PsiElement?